app/views/contact/_contact-link.html.erb

```diff
@@ -1,4 +1,4 @@
 <li>
-  <h2><%= link_to(link["Title"], link["URL"], rel: external_link?(link["URL"]) ? 'external' : '') %></h2>
-  <p><%= link["Description"] %></p>
+  <%= link_to(link["Title"], link["URL"], class: "govuk-link govuk-!-font-size-24 govuk-!-font-weight-bold", rel: external_link?(link["URL"]) ? 'external' : '') %>
```

Comment:
I think that this link might need an increase in line height:


Large screen:
![Screen Shot 2019-09-03 at 11 35 25](https://user-images.githubusercontent.com/1732331/64166501-23c6bf80-ce3f-11e9-9637-dc6dc95c5923.png)
---
Small screen:
![Screen Shot 2019-09-03 at 11 36 12](https://user-images.githubusercontent.com/1732331/64166506-275a4680-ce3f-11e9-9c04-f8d41a43a82f.png)

Reply:
updated to use local app classes